wiggum_app/static/js/scatterplot.js

```diff
@@ -5,6 +5,9 @@ var margin = {top: 30, right: 30, bottom: 30, left: 60},
 
 var scatterplot;
 
+// regular expression for invalid characters
```

Comment:
invalid for what? determined by where? add some notes for maintainability

Reply:
Added